core/cardano-sl-core.cabal

```diff
@@ -59,6 +59,10 @@ library
                        Pos.Util.Chrono
                        Pos.Util.Limits
 
+                       -- Update
+                       -- (Needed for wallet-new)
+                       Pos.Core.Update.Types
```

Comment:
It's re-exported from `Pos.Core.Update` module and I would prefer not to expose this module, but import `Pos.Core.Update` instead.
The reason why I find it better is that it allows us to change layout of modules in `Pos.Core.Update` without breaking the code outside `cardano-sl-core` as long as we re-export the same stuff. Now have 3 modules there, but later one of them may become too big and we may want to split it, for example. Or it can become too small and we will want to merge some modules. Or just rename for some reason.
There is a trade-off, because when you import `Pos.Core.Update` you depend on more stuff (because it contains more than just `Pos.Core.Update.Types`), that's why `cardano-sl-core` exposes not only `Pos.Core` (which re-exports virtually everything). I think current granularity (when `Pos.Core.Update` is exposed and `Pos.Core.Update.Types` is not) is a good balance between these trade-offs. `Pos.Core.Update` contains not too much stuff, but gives us some flexibility.

Reply:
Thank you very much for the review, @gromakovsky ! I genuinely not appreciated that module was exported via `Pos.Core.Update`, my bad! I agree with the general sentiment here, this way of exporting allows us to change things internally without breaking the external API. Leave it with me!